File: Modul_5/Soal_4/PRAK504-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>
int reverse(int angka){
    // Lengkapi Function ini
    int reverse = 0, sisa;
    while(angka != 0) {
        sisa = angka % 10;
        reverse = reverse * 10 + sisa;
        angka /= 10;
    }
    return reverse;
}

int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}

File: Modul_5/Soal_4/PRAK504-2310817210015-MuhammadBukhariFitri.py
def reverse(angka):
    # Lengkapi Function ini
    reverse = 0;
    while(angka != 0) :
        sisa = angka % 10;
        reverse = reverse * 10 + sisa;
        angka //= 10;
    return reverse;

input = input().split()
A, B = map(int, input)
A = reverse(A);
B = reverse(B);
C = A+B;
print(f"{reverse(C)}");